File: CS/T223550/Models/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace T223550.Models {
    public class Customer {
        public int ID { get; set; }
        public string Name { get; set; }
        public override string ToString() {
            return Name;
        }
    }
}

File: VB/Models/Customer.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace T223550.Models
	Public Class Customer
		Public Property ID() As Integer
		Public Property Name() As String
		Public Overrides Function ToString() As String
			Return Name
		End Function
	End Class
End Namespace
